### ISSUE TITLE
Can't compile 'Game'

### ISSUE THREAD

**Issue by L33B0B** [Da0ne/DZMods#102]
Hi,

Totally new to the whole dayz modding side of things. i have my own server and want to get this mod on it!

I've followed the steps to install this but i get this when trying to start it
![capture](https://user-images.githubusercontent.com/10597088/49330905-4a448b00-f58d-11e8-88a7-0106ffc2aff8.JPG)

I follwed the RPC install or at least i thought i did!!
Can anyone shed some light on this on how i fix it?

Thanks

**Comment by chawdinky** [comment]
This issue is because you didn't put the key in the keys folder or are missing files in your @RPCFramework. mod or didn't add the @RPCFramework mod with the @VanillPlusPlus. This issue is not a problem with this mod its just Human error.

**Comment by L33B0B** [comment]
I get it's human error. How do I correct it? Keys are in keys folder. Only bit I couldn't work out was step 3 of the rpc install.

**Comment by chawdinky** [comment]
Make sure you have all your files in the mod. Make sure command line has -mod=@RPCFramework;@VanillaPlusPlus

Here are the steps you must do to get it all working.

1. Download RPCFramework and VanillaPlusPlus from workshop
2. Put copy the folders @RPCFramework & @VanillaPlusPlus to your server folder
Example: G:\My Steam\Steam\SteamApps\common\DayZServer
![testserver](https://user-images.githubusercontent.com/14309490/49333743-d654b380-f589-11e8-9fa0-c9f2155dafaf.png)
3. Copy they "Keys" from the both mod folders to your "Keys" server folder
4. Add this line to your run.bat -mod=@RPCFramework;@VanillaPlusPlus
Example:  `start DayZServer_x64.exe -mod=@RPCFramework;@VanillaPlusPlus -config=serverDZ.cfg -port=2302 -noFilePatching`

**Comment by chawdinky** [comment]
After you have fixed this, please close this issue. Thanks ;)

**Comment by chawdinky** [comment]
PS.... Dont follow the RPCFramework install method. only add it into your mods..... Just follow my steps above nothing more nothing less.

**Comment by L33B0B** [comment]
I've done what you said and still getting the same error... :(

**Comment by chawdinky** [comment]
I would do a complete fresh install, you have managed to mess something up. This Error is because VanillaPlusPlus cant read the RPCFramework making VanillaPlusPlus GetRPCManager function broken. It cant find it/Undefined

**Comment by L33B0B** [comment]
i reinstalled the entire thing. followed the steps you give above and havent changed anything but IP's, names and passwords and still getting the same error. Any ideas?

**Comment by GravityWolfNotAmused** [comment]
Take the key out of the RPC and put the file inside of keys directory inside of root server directory.

**Comment by GravityWolfNotAmused** [comment]
Let me know if this works

**Comment by GravityWolfNotAmused** [comment]
@L33B0B I will be doing a fresh install with newest version when I get home, and will report things that I run into, and how to fix them.

**Comment by GravityWolfNotAmused** [comment]
@L33B0B I did a complete install of the game with all new game files. I got my server add-ons from the following: 
RPCManager - Steam WorkShop
VanillaPlusPlus - Github

All client side downloads came from Steam Workshop and should always come from there.

I followed the same install method as provided:
Put both add-ons on the server, All @ folders are located in server root, Keys need to be put into keys folder also located in server root, set startup command. Launch.

Client install:
Subscribe to both add-ons on the Steam Workshop.
Launch DayZ Launcher, Right click on RPCManager, Open in File, verify that you see both RPCManager and VanillaPlusPlus mods in !workshop.
Move keys into folders if haven't done so already for client.
Launch DayZ with launcher with mods enabled.

My server start up:
```
-config=serverDZ.cfg -profiles=stff -instanceid=1 -mod=@RPCFramework;@VanillaPlusPlus; -scrAllowFileWrite
```
Client start up:
```
-name=GravityWolf -mod=@VanillaPlusPlus;@RPCFramework 
```
